.github/ISSUE_TEMPLATE/orientation-epic.md

```diff
@@ -43,12 +43,14 @@ As a new VFS team member, I need to complete onboarding activities so I can succ
 - [ ] Review [Welcome](https://depo-platform-documentation.scrollhelp.site/getting-started/Welcome.1255276574.html) documentation
 - [ ] Review [Support Overview](https://depo-platform-documentation.scrollhelp.site/support/Support-overview.604012552.html) documentation
 - [ ] [Request Access to Tools](https://depo-platform-documentation.scrollhelp.site/getting-started/Request-access-to-tools.969605215.html)
+- [ ] Review the [Submitting a feature request](https://depo-platform-documentation.scrollhelp.site/support/Submitting-a-feature-request.1938915400.html)
```

Comment:
maybe we should reword this to "Review the guidance on _Submitting a feature request_"